src/pybind/mgr/k8sevents/module.py

```diff
@@ -345,19 +403,26 @@ def event_body(self):
 
     def write(self):
 
-        now=datetime.now(pytz.utc)
+        now=datetime.now(UTC())
```

Comment:
```
        now=datetime.utcnow()
```

Reply:
not the same. using pytz or the included simple class passes a tzinfo into the datetime object - which IIRC is expected by the timestamp when creating an event. I'll retest though